### PR TITLE
Changed news to blog in urls. Added blog to header. Accept HTML from …

### PR DESCRIPTION
**Pull request by Vindexus**
…markdown posts. Dates in post URLs.

### CODE REVIEW

File: src/js/deon-main.js [hunk 1]
```diff
@@ -1005,6 +1005,14 @@ function transformMarkdown (obj) {
   return marked(obj)
 }
 
+function transformMarkdownPost (obj) {
+  let html = marked(obj)
```
Comment (tmathews) [inline src/js/deon-main.js]: What is this for?

File: src/templates/blog.html [hunk 2]
```diff
@@ -85,17 +85,17 @@ <h3 class="subline">{{subline}}</h3>
...
             <div class="container cl-white text center">
-                <h1 class="title uppercase">{{title}}</h1>
+                <h1 class="title uppercase"><a href="{{link}}">{{title}}</a></h1>
                 <h3 class="subline">{{subline}}</h3>
                 <p class="date">{{date}}</p>
                 <ul class="blog-tags">
                     {{#tags}}
                         <li>
-                            <a class="button button--black" href="/news/tags/{{.}}">{{.}}</a>
+                            <a class="button button--black" href="/blog/tags/{{.}}">{{.}}</a>
```
Comment (tmathews) [inline src/templates/blog.html]: Why is it changing to blog?

File: src/js/deon-main.js (hunk 1 above)
Comment (Vindexus) [reply src/js/deon-main.js]: marked escapes html so we get `&lt;img src="` instead of `<img src="`

File: src/templates/blog.html (hunk 2 above)
Comment (Vindexus) [reply src/templates/blog.html]: Marketing team wanted it changed to blog.

Comment (tmathews) [reply src/templates/blog.html]: I wish they gave us a better reason than breaking all our existing URLs and current SEO. Can you get a clear reason from them?

File: src/js/deon-main.js (hunk 1 above)
Comment (tmathews) [reply src/js/deon-main.js]: I'm not sure this is the correct solution for this. Is it perhaps the way we are rendering it on the page and should be useing `{{{`?

Comment (Vindexus) [reply src/js/deon-main.js]: We already do that. This is some escaping that `marked` is doing, not Mustache.

File: src/templates/blog.html (hunk 2 above)
Comment (Vindexus) [reply src/templates/blog.html]: I'll ask them. All the old urls will still work, we have (blog|news) in the regex.

Comment (tmathews) [reply src/templates/blog.html]: Alright.

**Pull request merged**